### PR TITLE
Update gist-embed.js to add callback

### PR DESCRIPTION
**Pull request by jackycute**
Add callback to be called after ajax complete

### CODE REVIEW

File: gist-embed.js [hunk 1]
```diff
@@ -185,6 +185,9 @@
         },
         error: function(jqXHR, textStatus) {
           $elem.html('Failed loading gist ' + url + ': ' + textStatus);
+        },
+        complete: function() {
+            if(callback && typeof callback === 'function') callback();
```
Comment (blairvanderhoof) [inline gist-embed.js]: use 2 spaces instead of 4

Comment (blairvanderhoof) [inline gist-embed.js]: is checking the existence of callback before the typeof necessary?

**Comment by blairvanderhoof** [comment]
Thanks! Just a couple comments.

File: gist-embed.js (hunk 1 above)
Comment (jackycute) [inline gist-embed.js]: Not really, because it'll be 'undefined' if callback not exist which already covered with current statement.

**Comment by jackycute** [comment]
Fixed, Thanks a lot!

**Pull request merged**